cabal-install-solver/cabal-install-solver.cabal

```diff
@@ -39,18 +39,18 @@ library
   default-language: Haskell2010
   hs-source-dirs:   src
   hs-source-dirs:   src-assertion
-  ghc-options:      -Wall -fwarn-tabs -fwarn-incomplete-uni-patterns
-
-  if impl(ghc >=8.0)
```

Comment:
I think this and the other ghc compat stuff in this commit should be moved to the previous one, where the minimum ghc actually changed (or to a new commit)

Reply:
Oh, I see that file was `cabal-version: 2.2` even before the change in cabal-install. So I guess a new commit?

Reply:
Yes, it was changed upon exposing the new library to match the `cabal-install.dev` cabal file.